kedro-org/kedro-viz
# Fix/focus mode modular pipeline highlight

## Description

Resolves #921 

## Development notes

Added one more boolean property to `createInitialPipelineState` , to track when FocusModeIcon is hovered and create new action in the store for updating this value.

For the style I reused the class we already have for when node is clicked and all other nodes in the flowchart are pushed in the background with some opacity (class: `pipeline-node--faded`).

## QA notes
You can open the Gitpod link, and check the hover effect on FocusModeIcon in the left side node list.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1107"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
